src/components/Book.js

```diff
@@ -49,14 +54,14 @@ const Book = ({ book, handleRemoveBookClick }) => {
 
 Book.propTypes = {
   book: PropTypes.shape({
-    id: PropTypes.number.isRequired,
+    id: PropTypes.string.isRequired,
     genre: PropTypes.string.isRequired,
     title: PropTypes.string.isRequired,
     author: PropTypes.string.isRequired,
     completed: PropTypes.string.isRequired,
     chapter: PropTypes.string.isRequired,
   }).isRequired,
-  handleRemoveBookClick: PropTypes.func.isRequired,
+  // handleRemoveBookClick: PropTypes.func.isRequired,
```

Comment:
- [ ] Kindly remove commented code. It can make your code harder to read. You can use git history to view old code.